src/Tec/Plugin.php

```diff
@@ -184,16 +175,85 @@ public function get_all_options() {
 	/**
 	 * Get a specific extension option.
 	 *
-	 * @param $option
+	 * @param        $option
 	 * @param string $default
 	 *
 	 * @return array
-	 *
-	 * TODO: Remove if not using settings
 	 */
 	public function get_option( $option, $default = '' ) {
 		$settings = $this->get_settings();
 
 		return $settings->get_option( $option, $default );
 	}
+
+	/**
+	 * Apply a fieldset to a newly created RSVP or ticket.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param $ticket   The ticket object with all its data.
+	 * @param $data     The ticket data sent.
+	 *
+	 * @param $post_id  The ID of the post / event for which the RSVP / ticket is being created.
```

Comment:
The parameters should be in order. Then, the parameter should have the param type, name of the variable and a short description. For example:

`@param int $post_id  The ID of the post / event for which the RSVP / ticket is being created.`

```suggestion
	 * @param int $post_id  The ID of the post / event for which the RSVP / ticket is being created.
	 * @param Tribe__Tickets__Ticket_Object $ticket  The ticket object with all its data.
	 * @param array $data     The ticket data sent.

```

Reply:
^ I'd re-check the indentation here and if the param types are correct, but something like that should do it.

Reply:
Great, thanks!
The `$post_id` was moved to the end by PHP Storm's code formatting. I guess because the `$post_id` is not being used in the method.